examples/hello-world/architect.yml

```diff
@@ -9,7 +9,8 @@ keywords:
 
 services:
   api:
-    image: heroku/nodejs-hello-world
```

Comment:
Looks like there are other places we use this image in the cli. Search `heroku/nodejs-hello-world`